infra/terraform/gcp/modules/ComputeInstance/mongod-cfgsvr-1.tf

```diff
@@ -8,8 +8,10 @@ resource "google_compute_instance" "mongod_cfgsvr_1" {
     "env"  = "prod"
     "type" = "mongo-config-server"
   }
+  
+  allow_stopping_for_update = true
 
-  machine_type = "e2-highmem-2"
+  machine_type = "e2-standard-2"
```

Comment:
[downgrade config servers to 'e2-standard-2'](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/4ba14dcab7cf21af03ad7b91995cb13bf0659bcb)